src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/validation/MetafileJsonValidatorTest.java

```diff
@@ -54,6 +65,40 @@ public void should_parse_envelope_data_with_no_non_scannable_items_in() throws I
         assertThat(envelope.getPayments()).hasSize(1);
     }
 
+    @Test
+    public void should_parse_envelope_data_with_no_account_details_for_postal_payment() throws IOException {
+        Envelope envelope = getEnvelope("/metafiles/valid/postal-payment-method.json");
+
+        assertThat(envelope.getPayments()).hasSize(1);
+        assertThat(envelope.getPayments().get(0).getAmount()).isEqualTo(100.0);
```

Comment:
Have you considered scenario where you may have two payment types for e.g solicitor may send cheque and postal order(worth checking with Ade)?
get(0) will fail if you have two payment methods in a single envelope.